demos/index.html

```diff
@@ -41,8 +41,14 @@
 
             <nav class="gpii-fd-nav">
                 <div class="gpiic-fd-navButtons gpii-fd-navButtons">
-                    <button id="gpiic-fd-navButtons-back" class="left gpii-fd-navButtons-back" tabindex="1"><span class="gpii-fd-icon"></span>back</button>
-                    <button id="gpiic-fd-navButtons-next" class="right gpii-fd-navButtons-next" tabindex="1">next<span class="gpii-fd-icon"></span></button>
+                    <button id="gpiic-fd-navButtons-back" class="left gpii-fd-navButtons-back" tabindex="1">
+                        <span class="left gpii-fd-icon"></span>
+                        <span class="gpiic-fd-navButtons-backLabel">back</span>
```

Comment:
Foundation seems to really mess up the styling if I use a label element here. I'm trying to overcome it, but I can't figure out where it's happening. I wonder if it's worth it?